plugins/amdgpu/amdgpu_plugin.c

```diff
@@ -1275,9 +1343,92 @@ static int restore_process(int fd, CriuKfd *e)
 	return ret;
 }
 
+static int restore_devices(int fd, CriuKfd *e)
```

Comment:
Can you put a comment above to describe briefly what restore device does specifically since the name is generic.